NMS-16255: News Feed configuration documentation (#6978)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/admin/configuration/newsfeed-configuration.adoc

= News Feed Configuration

:description: Learn how to configure the News Feed service in {page-component-title}.

The News Feed is a panel on the main {page-component-title} page which displays information from the OpenNMS website RSS news feed. This can help keep users informed of the latest OpenNMS and {page-component-title} news.

The {page-component-title} server makes REST queries to the https://www.opennms.com/feed/[OpenNMS RSS feed] to get the latest feed data.

This feature is enabled by default, but can be disabled using the configuration below.
Disabling will remove the panel from the front page and the server will no longer make queries for the feed.

There are two configuration points in the `etc/opennms.properties` file.

[options="autowidth"]
|===
| Property  | Description   | Default Value

| opennms.newsFeedPanel.show
| Whether to show the News Feed panel. Need to explicitly set to `false` to disable.
| true

| opennms.newsFeedPanel.url

> | The URL of the RSS feed used to get the feed data. Generally this should be commented-out, but could be set for testing purposes, or to set a proxy URL, etc.

Reviewer: ```suggestion
| The URL of the RSS feed used to get the feed data.
This will default to the OpenNMS feed if omitted.
```
